docs/src/main/asciidoc/opentelemetry-dev-services.adoc

```diff
@@ -0,0 +1,41 @@
+////
+This guide is maintained in the main Quarkus repository
+and pull requests should be submitted there:
+https://github.com/quarkusio/quarkus/tree/main/docs/src/main/asciidoc
+////
+= Dev Services for OpenTelemetry
+:extension-status: preview
+include::_attributes.adoc[]
+:categories: data
+:summary: Start OpenTelemetry automatically in dev and test modes.
+
+Quarkus supports a feature called Dev Services that allows you to run collect traces without any config.
+What that means practically, is that if you have docker running and have not configured `quarkus.otel.exporter.otlp.traces.endpoint`,
+Quarkus will automatically start a Jaeger container when running tests or dev mode, and automatically configure the connection.
+
+Available properties to customize the OpenTelemetry DevService.
+
+include::{generated-dir}/config/quarkus-opentelemetry-config-group-devservices-dev-services-config.adoc[opts=optional, leveloffset=+1]
+
+When running the production version of the application, the exporter connection needs to be configured as normal,
+so if you want to include a production tracing config in your `application.properties` and continue to use Dev Services
+we recommend that you use the `%prod.` profile to define your exporter configuration settings.
+
+Dev Services for OpenTelemetry relies on Docker to start the server.
+If your environment does not support Docker, you will need to start the server manually, or connect to an already running server.
+
+NOTE: If you want to disable ryuk container, set variable `TESTCONTAINERS_RYUK_DISABLED=true`.
+
+== Shared server
+
+Most of the time you need to share the server between multiple applications.
+Dev Services for OpenTelemetry implements a _service discovery_ mechanism for your multiple Quarkus applications running in _dev_ mode to share a single server.
+This results in having _all_ traces from all traced microservices in the same container. Even after the original microservice is
+shut down, the container persists. This can be changed with setting `testcontainers.reuse.enable=false` under `~/.testcontainers.properties` file.
```

Comment:
You added `shared` configuration property to disable sharing of containers, so I presume only case when users should use TestContainers properties directly is when they want to use shared container, but cut it down when original Quarkus application is shut down?

Reusable containers is still experimental feature and if we enable it here by default, it can come to some unexpected behavior when running in CI. IMHO _Most of the time you need to share the server between multiple applications._ is strong statement. How often you test OpenTelemetry with more than one Quarkus application and how do you automate it (don't get me wrong, I know it is possible, I did it just last Friday, but I don't agree it is common case).

Reply:
What I try to say is that documentation is giving user incorrect expectations, let me give you an example. Dev Services for PostgreSQL also set reusable `io.quarkus.devservices.postgresql.deployment.PostgresqlDevServicesProcessor#setupPostgres`, however whenever I run it, I can see logged

```bash
18:03:47,642 INFO  [app] 18:03:44,353 Reuse was requested but the environment does not support the reuse of containers
18:03:47,643 INFO  [app] To enable reuse of containers, you must set 'testcontainers.reuse.enable=true' in a file located at /home/mvavrik/.testcontainers.properties
```

that's by default, I don't have any local TC properties file

Reply:
ah, okay I do, not sure how it got there :-/ my bad!